docs/release-notes/NuGet-5.9.md

```diff
@@ -13,12 +13,13 @@ NuGet distribution vehicles:
 
 | NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
 |:---|:---|:---|
-| [**5.9**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.9](https://visualstudio.microsoft.com/downloads/) | [5.0](https://dotnet.microsoft.com/download/dotnet-core/5.0)<sup>1</sup> |
+| [**5.9.0**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.9](https://visualstudio.microsoft.com/downloads/) | [5.0.200](https://dotnet.microsoft.com/download/dotnet-core/5.0)<sup>1</sup> |
+| [**5.9.1**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.9](https://visualstudio.microsoft.com/downloads/) | [5.0.202](https://dotnet.microsoft.com/download/dotnet-core/5.0)<sup>1</sup> |
 
 <sup>1</sup> Installed with Visual Studio 2019 with .NET Core workload
   
 > [!NOTE]
-> Visual Studio 16.9, MSBuild 16.9, and .NET 5.0.3+ requires NuGet.exe 5.9 or later.
```

Comment:
Why did .NET 5.0.3 go down to 5.0.2?

Reply:
It's incorrect. 
5.0.3 will be released with 5.10.

Reply:
ugh we are getting bit by the sdk vs. runtime version thing here. I think we're using the runtime version:

![image](https://user-images.githubusercontent.com/1288848/114624623-9c763a80-9c76-11eb-9a46-7f894b7dfb29.png)

We should likely standardize on the SDK version and include `SDK 5.0.200+` rather than the current runtime.

Reply:
Proposed change:

`.NET SDK 5.0.200+`

Reply:
fixed.